Purchases/FoundationExtensions/NSDate+RCExtensions.m

```diff
@@ -10,7 +10,7 @@
 
 @implementation NSDate (RCExtensions)
 
-- (UInt64)millisecondsSince1970 {
+- (UInt64)rc_UInt64MillisecondsSince1970 {
```

Comment:
also added the `UInt64` bit to make the return time unambiguous, I'll update the one in `purchases-hybrid-common` as well.

Reply:
just a personal preference but i think the type at the end is a little more readable. rc_millisecondsSince1970asUInt64 or maybe even rc_millisecondsSince1970_UInt64?  🤷

Reply:
I actually went back and forth on naming it like 5 times 😂 I agree, I'll go back to rc_millisecondsSince1970AsUInt64

Reply:
@beylmk updated